pages/instructorAssessmentInstance/instructorAssessmentInstance.sql

```diff
@@ -53,6 +53,7 @@ WHERE
 -- BLOCK select_instance_questions
 SELECT
     iq.*,
+    to_jsonb(iq.modified_at) AS modified_at,
```

Comment:
I don't actually understand what this is doing - why `to_jsonb`?

Reply:
I forgot to make a comment on this. If `modified_at` is used as is, it will be interpreted by JS as a `Date` object, which will print the value (using regular methods like `toISOString()`) with millisecond precision. The problem is that PostgreSQL is checking the date with microsecond precision. This conversion ensures the date is converted to a string format that will be interpreted and converted back to timestamptz with the appropriate values when the data is passed back through the POST request.

I used `to_jsonb` because I noticed that the precision problem didn't happen for the middleware query, which returns a JSONB object for the instance question. If you know of a better alternative I'm happy to consider it.

Reply:
Ahh, that makes perfect sense. I wonder if `CAST(iq.modified_at AS TEXT)` would make the intent a little clearer? I don't feel strongly about that though. But with either this or `to_jsonb`, an inline comment would be helpful 👍

Reply:
Ah, right, that looks clearer. I'll make the changes and add a comment.